core/dbt/exceptions.py

```diff
@@ -281,6 +281,17 @@ def __init__(self, message, project=None, result_type='invalid_project'):
         self.result_type = result_type
 
 
+class FailFastException(Exception):
```

Comment:
Hmm, that's an interesting question. Maybe derive from `RuntimeException` and skip `__init__`/`__str__`? It would be nice to attach the failed node to the exception, if we can!

Reply:
So when raising it, something like this (if it works - I have not tested literally anything about it!):
`raise FailFastException(result.error, node=getattr(result, 'node', None)`

Reply:
Ok. Thanks

Reply:
Done 😉